Framework/Source/Effects/SkyBox/SkyBox.cpp

```diff
@@ -65,9 +65,9 @@ namespace Falcor
         return createFromTexture(skyBox, serializer.getValue("loadAsSrgb").b, Sampler::create(samplerDesc));
     }
 
-    RenderPassSerializer SkyBox::serialize(const std::shared_ptr<RenderPass>& pRenderPass)
+    RenderPassSerializer SkyBox::serialize()
```

Comment:
Pass the serializer in as a ref. I didn't think much about its design, but it's probably a single object that the graph exporter passes around (something like a ofstream)